pkg/parca/parca.go

```diff
@@ -584,10 +584,7 @@ func runScraper(
 		return err
 	}
 
-	externalLabels := labels.Labels{}
-	for name, value := range flags.ExternalLabel {
-		externalLabels = append(externalLabels, labels.Label{Name: name, Value: value})
-	}
+	externalLabels := labels.FromMap(flags.ExternalLabel)
```

Comment:
For reference: https://github.com/prometheus/prometheus/blob/2d588725c12e46e4e71da3899ec05bb933151021/model/labels/labels.go#L367